application/src/test/java/uk/gov/hmcts/net/ccd/definition/store/BaseTest.java

```diff
@@ -49,7 +49,7 @@
 })
 @TestPropertySource(locations = "classpath:test.properties")
 public abstract class BaseTest {
-    public static final String EXCEL_FILE_CCD_DEFINITION = "/CCD_TestDefinition_V29_RDM-1831.xlsx";
+    public static final String EXCEL_FILE_CCD_DEFINITION = "/CCD_TestDefinition_V30.xlsx";
```

Comment:
how come we're not adding anymore the story number to the file name? that proved useful to avoid collisions when multiple persons working on it at the same time